packages/unlock-js/src/walletService.ts

```diff
@@ -1102,4 +1107,31 @@ export default class WalletService extends UnlockService {
       purchaseMessage,
     }
   }
+
+  /**
+   * Performs a purchase using the CardPurchaser contract
+   * @param param0
+   * @returns
+   */
+  async purchaseWithCardPurchaser({
+    transfer,
+    purchase,
+    callData,
+  }: PurchaseWithCardPurchaserParams) {
```

Comment:
is this consistent with the argument format we use in above methods?

Reply:
yes